lightning-persister/src/util.rs

```diff
@@ -135,7 +135,7 @@ mod tests {
 		// Create the channel data file and make it a directory.
 		fs::create_dir_all(get_full_filepath(path.clone(), filename.to_string())).unwrap();
 		match write_to_file(path.clone(), filename.to_string(), &test_writeable) {
-			Err(e) => assert_eq!(e.kind(), io::ErrorKind::Other),
+			Err(e) => assert_eq!(e.raw_os_error(), Some(libc::EISDIR)),
```

Comment:
Consider adding a `TODO` to switch back to `kind` once possible. Though I guess that is a long way out. 😕

Reply:
Yea, that's gonna require at least rustc-now, so I guess in at least two years per the rust-bitcoin MSRV policy, assuming distros upgrade. I think this is fine, in any case, at least it won't break.